backend/client.js

```
@@ -166,7 +166,7 @@ router.post("/", async function (req, res, next) {
       message.push("Could not connect to the Hue bridge while requesting smart scenes");
     });
 
-  var url = "https://plex.tv/api/users";
+  var url = "https://clients.plex.tv/api/users";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'url'. Consider to delete it or rename it.

'url' is defined here:


(lint/suspicious/noRedeclare)